mlfinlab/feature_importance/importance.py

```diff
@@ -60,7 +61,7 @@ def mean_decrease_impurity(model, feature_names):
     return importance
 
 
-def mean_decrease_accuracy(model, X, y, cv_gen, sample_weight_train=None, sample_weight_score=None, scoring=log_loss):
+def mean_decrease_accuracy(model, X, y, cv_gen, clustered_subsets=None, sample_weight_train=None, sample_weight_score=None, scoring=log_loss):
```

Comment:
Can we also introduce random_state:int = 42 (this is needed tor feature importance results being reproducible)

Reply:
That is a very good point. That will be done.